web/static.go

```diff
@@ -121,3 +139,27 @@ func unsupportedBrowserScriptHandler(w http.ResponseWriter, r *http.Request) {
 	templatesDir, _ := templates.GetTemplateDirectory()
 	http.ServeFile(w, r, filepath.Join(templatesDir, "unsupported_browser.js"))
 }
+
+func getOpenGraphMetaTags(c *Context, originalHTML string, titleTemplate string) string {
+	modifiedHTML := originalHTML
+	siteName := c.App.Srv().Config().TeamSettings.SiteName
+	siteDescription := model.TeamSettingsDefaultCustomDescriptionText
+	customSiteDescription := c.App.Srv().Config().TeamSettings.CustomDescriptionText
+	if customSiteDescription != nil && *customSiteDescription != "" {
+		siteDescription = *customSiteDescription
+	}
+
+	if siteDescription == "" {
+		return ""
+	}
```

Comment:
This won't ever be the case, given the code above. Also, we wouldn't want to return an empty string here, since that would end up removing the `title` tag in the caller function

Reply:
You are right, I tried to code on my phone.. So I just missed it.